src/components/crossForm/CrossForm.vue

```diff
@@ -1,5 +1,12 @@
 <template>
-  <Form id="crossForm" ref="crossForm" name="crossForm" class="cross-form" @submit="onSubmit">
+  <Form
+    v-if="false"
```

Comment:
no need of v-if if it's never gonna be true

Reply:
removed